libp2p_port/internal/port/port.go

```diff
@@ -0,0 +1,49 @@
+package port
```

Comment:
maybe move to `native/libp2p_port`?

Reply:
As this is not a NIF (Native implemented function) I added the "port" reference to the directory instead. This code is not dynamically linked with our code, so I thought calling it "native" was a bit confusing. I can move it to the native folder if you still think that's more appropriate though!